lighthouse-core/lib/log.js

```diff
@@ -129,7 +129,40 @@ class Log {
     Log.events.issueStatus(title);
     return Log._logToStdErr(`${title}:verbose`, Array.from(arguments).slice(1));
   }
+
+  /**
+   * Add surrounding escape sequences to turn a string green when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static greenify(str) {
+    return `${Log.green}${str}${Log.reset}`;
+  }
+
+  /**
+   * Add surrounding escape sequences to turn a string red when logged.
+   * @param {string} str
+   * @return {string}
+   */
+  static redify(str) {
+    return `${Log.red}${str}${Log.reset}`;
+  }
+
 }
+
+Log.green = '\x1B[32m';
```

Comment:
no not exports :)

but not my call to make so waiting for the rest of the team to give final feedback

```
const red = '\x1B[31m';
const reset = '\x1B[0m';

class Log {
...
static redify(str) {
  return `${red}${str}${reset}`;
}
module.exports = Log;
```